gpfit/tests/t_examples.py

```diff
@@ -48,9 +46,14 @@ def test_hoburgabbeel_ex6_1(self, example):
             setattr(TestExamples, name, old_test)  # move to a non-test fn
             delattr(TestExamples, fn)  # delete the old old_test
             new_name = "test_%s" % (name)
-            setattr(TestExamples, new_name, new_test(name, None))
+            setattr(TestExamples,
+                    new_name,
+                    logged_example_testcase(name=name,
+                                            imported=IMPORTED_EXAMPLES,
+                                            exampledir=EXAMPLE_DIR))
     TESTS.append(TestExamples)
```

Comment:
@bqpd: the 16 lines above are replicated almost exactly in gpkit/tests/t_examples.py. The only difference is `logged_example_testcase` vs `new_test`.

Is there a better way to do this? somehow include more of this in the `logged_example_testcase` method over at GPkit? Make a new class that inherits from `unittest.TestCase` and handles all out fancy example logging tricks?